packages/charts/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxis.base.tsx

```diff
@@ -84,7 +84,7 @@ export class HorizontalBarChartWithAxisBase extends React.Component<
       color: '',
       dataForHoverCard: 0,
       isCalloutVisible: false,
-      isLegendSelected: false,
+      isLegendSelected: props.legendProps?.selectedLegend?.length! > 0,
```

Comment:
What if the legend title text is '' (empty string)? In that case even if the legend is selected, isLegendSelected will be false

Reply:
if legend title text is empty string, that means no legend is selected.

Reply:
Yes, correct.